source/install/test_cc.sh

```diff
@@ -21,6 +21,6 @@ make install
 mkdir -p ${BUILD_TMP_DIR}/exec_tests
 cd ${BUILD_TMP_DIR}/exec_tests
 
-${INSTALL_PREFIX}/bin/runUnitTests_lib
-${INSTALL_PREFIX}/bin/runUnitTests_cc
+#${INSTALL_PREFIX}/bin/runUnitTests_lib
+#${INSTALL_PREFIX}/bin/runUnitTests_cc
```

Comment:
Why are the test for lib and cc commented out ?

Reply:
Typo - I want to make local tests faster but don't want to push it.